src/axom/primal/operators/split.hpp

```diff
@@ -78,6 +78,58 @@ void split(const Octahedron<Tp, NDIMS>& oct,
   out.push_back(Tet(oct[Q], oct[S], oct[U], C));
 };
 
+/*!
+ * \brief Splits an Octahedron into eight Tetrahedrons
+ *
+ * \tparam Tp the coordinate type, such double or float
+ * \tparam NDIMS the number of spatial dimensions (must be 3).
+ * \param [in] oct The Octahedron to split
+ * \param [out] out C array of 8 Tetrahedron objects; the fragments of
```

Comment:
```suggestion
 * \param [out] outPtr C array of 8 Tetrahedron objects; the fragments of
```
What is a "C array"? Is this a typo?

Reply:
I think of it as a C pointer to memory with enough space for 8 tets.

Reply:
Done.